core/src/main/java/pl/agh/edu/actor/utils/CustomLabel.java

```diff
@@ -1,16 +1,40 @@
 package pl.agh.edu.actor.utils;
 
+import com.badlogic.gdx.graphics.g2d.Batch;
 import com.badlogic.gdx.scenes.scene2d.ui.Label;
 import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
 
 import pl.agh.edu.actor.GameSkin;
 
-public abstract class CustomLabel extends Label {
+public class CustomLabel extends Label {
+	private boolean isUnderscore = false;
+
 	public CustomLabel(String font, String backgroundPatch) {
+		this(font);
+		getStyle().background = new NinePatchDrawable(GameSkin.getInstance().getPatch(backgroundPatch));
+	}
+
+	public CustomLabel(String font) {
 		super("", GameSkin.getInstance());
 		LabelStyle labelStyle = new LabelStyle();
 		labelStyle.font = GameSkin.getInstance().getFont(font);
-		labelStyle.background = new NinePatchDrawable(GameSkin.getInstance().getPatch(backgroundPatch));
 		this.setStyle(labelStyle);
 	}
+
+	public void setUnderscore(boolean isUnderscore) {
```

Comment:
jeżeli chcemy mieć możliwość ustawiana koloru tego underscora to jak dla mnie ta metodka może mieć tylko arguent odnośnie koloru